### ISSUE TITLE
Feature Request: Spatial Audio Control

### ISSUE THREAD

**Issue by antgiant** [theta360developers/4k-long-video#1]
I love this plugin, but spatial audio is important to me, and I am not an Android developer. So adding it back in would be a significant challenge. Would you be willing to add it in as a toggle-able option?

Something like the [Shutter Volume Control](https://pluginstore.theta360.com/plugins/guide.theta360.shuttervolumecontrol/) plugin where it can be turned on or off with the wireless button would be great! Extra great if it could also remember the last setting so that I wouldn't have to turn it on every time I enable the plugin.

Thanks again for your work on this!

**Comment by jcasman** [comment]
@antgiant - Is getting ahold of the .wav file separately and then adding it to the video file in something like Premier Pro an option? There are instructions here for grabbing the .wav file: https://community.theta360.guide/t/theta-v-long-video-plug-in-recording-beyond-25-minutes/3483  (scroll down to "Transfer Spatial Audio File to Your Computer" section)

**Comment by antgiant** [comment]
@jcasman Yes I’m willing to combine the files, I expected to do that actually. (By the way Facebook now provides a free tool that can do it as well for those of us without Premier Pro. [Spatial Workstation](https://facebook360.fb.com/spatial-workstation/))

However, if you read to the end of your referenced chain or the note on the official version’s description you will note that the wave file is now auto deleted. Hence my request for an option to get the file that doesn’t also require me to become an Android developer.
